etc/devices/redfishpower-cray-r272z30.dev

```diff
@@ -54,7 +52,9 @@ specification "redfishpower-cray-r272z30" {
 		expect "redfishpower> "
 	}
 	script cycle_ranged {
-		send "cycle %s\n"
+		send "off %s\n"
+		expect "redfishpower> "
+		send "on %s\n"
 		expect "redfishpower> "
```

Comment:
Usually we put a delay in here?

Reply:
lol, i guess the one i copied and pasted from is missing a delay.  I'll fix up everywhere.

Reply:
i just remembered that delay isn't strictly necessary here because we "wait until off" for the off command.  But a couple seconds delay couldn't hurt since that is the common pattern.

Reply:
The delay is to actually have the hardware in the off state for a period of time before reapplying power.  The "wait until on|off" thang is presumed to be how things work throughout powerman.